### ISSUE TITLE
Subfolders instead of only-.Rmd documents for different pages

### ISSUE THREAD

**Issue by t-rop** [soztag/fossos#130]
To avoid having to specify *.html at every link and for easier maintenance in the future, pages like stack should not be in \fossos\stack.Rmd but in \fossos\stack\index.html

**Comment by maxheld83** [comment]
@t-rop I just improved the highlighting a bit in the above.

**Comment by maxheld83** [comment]
I agree that `fossos\stack\index.html` is a much nicer url, partly because that also works as `fossos\stack` and it's just generally more idiomatic.
It might need a bit of research to figure out how to get this work from the rmarkdown website, because AFAIK, Rmarkdown projects require all `*.Rmd`s to be in the project root, otherwise all working directory hell breaks loose.
But there's probably a way to do this.

**Comment by maxheld83** [comment]
this would be accomplished by #201
